microsoft-edge/webview2/concepts/process-related-events.md

```diff
@@ -0,0 +1,190 @@
+> This document is a high-level overview of the aforementioned events. Ultimately, the reference documentation for each individual API mentioned in this document shall prevail.
+
+## WebView2 Process Model and related APIs
+The [WebView2 Process Model](process-model.md) uses the WebView2 Runtime to support your WebView2 application. Because most of the magic in the WebView2 controls in your application happens out-of-process (in the WebView2 Runtime), failures and general information about these processes are passed to your application through different WebView2 APIs. The APIs used to report this information to your application include:
+
+  * CoreWebView2
+    - `BrowserProcessId` property
+    - `ProcessFailed` event
+  * CoreWebView2Environment
+    - `BrowserProcessExited` event
+    - `ProcessInfosChanged` event
+    - `GetProcessInfos` method
+
+  > [!NOTE]
+  > This is an overview at the conceptual level. For detailed API names and related types in your app's framework, see [Process Management](overview-features-apis.md#process-management).
+
+In this document, we discuss how to handle process exits and failures, using the `CoreWebView2.ProcessFailed` and `CoreWebView2Environment.BrowserProcessExited` events.
+
+## Process Events
+The WebView2 Runtime is composed of different types of processes, including:
+  * Main browser process
+  * Renderer processes
+  * Utility processes
+  * GPU process
+
+  > [!NOTE]
+  > This is an illustrative and incomplete list of process kinds. The purpose and management details for these processes is part of the Chromium architecture and beyond the scope of this document, or the WebView2 architecture.
+
+When you create and initialize a WebView2 control, WebView2 will ensure there's a WebView2 Runtime to power your control and connect to its [WebView2 Process Group](process-model.md#processes-in-the-webview2-runtime). Once this connection is established, your control will start monitoring these processes for the following events:
+
+  * **Main browser process exits.** If the main browser process exits for _any reason_, the `CoreWebView2Environment` will raise the `BrowserProcessExited` event. Use this event to synchronize operations involving the WebView2 Runtime resources and lifetime, such as _User Data Folder_ management and updates. See [Main Browser Process Exits](<link>) below for more details.
+  * **Any process failure.** When _any of the processes_ in the WebView2 Runtime fails, the CoreWebView2 will raise the `ProcessFailed` event. Use this event for diagnostics and recovery from failures in the WebView2 processes. See [Process Failures](<link>) below for more details.
+
+  > [!NOTE]
+  > There is some overlap between these two events. For example, a main browser process crash will produce both a `ProcessFailed` event and a `BrowserProcessExited` event, since the main browser process _exited_ because of a failure. See [Main Browser Process Crashes](<link>) for more information about this case.
```

Comment:
```suggestion
  > There is some overlap between these two events. For example, a main browser process crash will produce both a `ProcessFailed` event and a `BrowserProcessExited` event, since the main browser process _exited_ because of a failure. See [Main browser process crashes](#main-browser-process-crashes) below for more information about this case.
```
i strongly prefer "below" or "above", to indicate a local link in same page